mapper/mapper_test.go

```diff
@@ -310,6 +310,7 @@ func TestReleaseCalendarMapper(t *testing.T) {
 			So(calendar.Pagination.TotalPages, ShouldEqual, 3)
 			So(calendar.Pagination.CurrentPage, ShouldEqual, 1)
 			So(calendar.Pagination.Limit, ShouldEqual, 5)
+			So(calendar.TotalSearchPosition, ShouldEqual, 0)
```

Comment:
This test covers the base case of being on page 1. If the tests are to serve as a specification of how the system should behave, then there should also be test cases that exercise the calculation `(currentPage - 1) * itemsPerPage`, demonstrating the relationship that exists between these variables.

Reply:
i have introduct more test to capture your suggections.